pkg/scheduler/util/priorities/image_locality.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
move this into plugin directory; do not put plugin specific util in this directory.